backend/infrahub/core/changelog/models.py

```diff
@@ -228,6 +228,10 @@ def updated_fields(self) -> list[str]:
         """Return a list of update fields i.e. attributes and relationships"""
         return list(self.relationships.keys()) + list(self.attributes.keys())
 
+    @property
+    def has_changes(self) -> bool:
+        return bool(self.updated_fields)
```

Comment:
Nit pick but I think `len(self.updated_fields) > 0` is clearer

Reply:
I think that the original is perhaps a bit more Pythonic, but either or is fine with me. I've pushed an update.